frontend/actions/actions.js

```diff
@@ -0,0 +1,20 @@
+const AppDispatcher = require("../dispatcher/dispatcher");
```

Comment:
Even better, this webpack contributor [recommends](https://github.com/webpack/webpack/issues/472#issuecomment-55706013) using [resolve.root](https://webpack.github.io/docs/configuration.html#resolve-root)